requirements.txt

```
@@ -1,4 +1,4 @@
-Django==4.1.13
+Django==5.0.2
```

```suggestion
Django==5.0.3
```
https://docs.djangoproject.com/en/5.0/releases/5.0.3